src/main/scala/zio/nio/ByteBuffer.scala

```diff
@@ -3,7 +3,7 @@ package zio.nio
 import zio.{ Chunk, IO, UIO, ZIO }
 import java.nio.{ BufferUnderflowException, ByteOrder, ReadOnlyBufferException, ByteBuffer => JByteBuffer }
 
-class ByteBuffer protected[nio] (private val byteBuffer: JByteBuffer) extends Buffer[Byte](byteBuffer) {
+class ByteBuffer protected[nio] (protected[nio] val byteBuffer: JByteBuffer) extends Buffer[Byte](byteBuffer) {
```

Comment:
why you need this?

Reply:
In method receive in DatagramChannel I need to have access to the original byte buffer which was not the case when this was private.